modules/cluster-logging-deploy-console.adoc

```diff
@@ -10,7 +10,7 @@ You can use the {product-title} web console to install the OpenShift Elasticsear
 
 [NOTE]
```

Comment:
This references the `{product-title} web console` - do you think it needs conditionals to specify  `{hybrid-console}` or `{cluster-manager-url} {hybrid-console}` for OSD and ROSA?

Reply:
I think this on also applies to any of the console references in the logging docs